ui/src/components/Preview/index.jsx

```diff
@@ -1,45 +1,78 @@
+import classNames from 'classnames';
+import { string, func } from 'prop-types';
 import React, { Component } from 'react';
 import Markdown from '@mozilla-frontend-infra/components/Markdown';
-import ListItem from '@material-ui/core/ListItem';
 import Tab from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
 import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { THEME } from '../../utils/constants';
 
+@withStyles(theme => {
+  const borderColor =
+    theme.palette.type === 'light'
+      ? fade(THEME.BLACK, 0.23)
+      : fade(THEME.WHITE, 0.23);
+
+  return {
+    tab: {
+      flexGrow: 1,
+    },
+    tabContent: {
+      marginTop: theme.spacing.unit,
+      padding: theme.spacing.unit,
+    },
+    markdownContainer: {
+      borderColor,
+      borderWidth: 1,
+      borderRadius: theme.shape.borderRadius,
+      borderStyle: 'solid',
+    },
+  };
+})
 export default class Preview extends Component {
+  static propTypes = {
+    onValueChange: func,
+    value: string,
+    placeholder: string,
+  };
+
   state = {
-    value: 0,
+    tabValue: 0,
```

Comment:
Rename to `tabIndex`. Additionally, let's have a `defaultTabIndex` so that users can pick which tab to show first.

Reply:
For the hook form, let's show the preview tab by default.